site/_posts/2022-06-28-remix-demo.md

```diff
@@ -0,0 +1,55 @@
+
+File-based session storage has one drawback that is only relevant at scale: once you need more than one Remix server, you will have to pin each user to a particular server for the duration of that session. However most developers are unlikely to encounter that kind of problem -- or by the time you do, you'll have the resources to solve it.
+
+You will also need a working FusionAuth instance. The easiest way to get this for testing purposes is to have a short conversation with a member of our [Sales team](https://fusionauth.io/contact) who will be able to provision a cloud instance for you. The second fastest way is to use the Docker install. The third fastest way is to use the Fast Path install detailed in our [Five Minute Setup Guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide). In all cases you should install the development data included in our [Kickstart project](https://github.com/FusionAuth/fusionauth-example-kickstart/tree/master/fusionauth).
+
+Once you have a FusionAuth instance, you should be able to go to the admin page and configure your FusionAuth instance like this:
+
+<img src="/assets/img/blogs/connecting-fusionauth-remix/fusionauth-config.png" />
+
+Then you must copy some values to `env.example` and change the name of the file to .env to set environment variables. At this point you should be ready to fire up the Remix server and try out the example.
+
+## Using FusionAuth with Remix
+
+Once you've done all this, auth will be trivially easy -- because all the hard lifting is being done by FusionAuth. Navigate to `http://localhost:3000` and click the login link. That will redirect you to the FusionAuth login -- which you can customize easily with your own CSS! -- and if you loaded the Kickstart data as directed you should now be able to log in as `dinesh@fusionauth.io` with password `password`. You will be redirected to `auth/callback` and from there dumped into `dashboard`. Try the same clickpath again and see what happens! If you want to make sure this was not a fluke, try the `/logout` route.
+
+Literally every time you need to add authenticatiion to a view, you merely call `authenticator.authenticate` with a successRedirect and a failureRedirect as shown in `app/routes/auth.callback.tsx`. Yes it really is that simple to get the best authentication in the business!
```

Comment:
Would be awesome to except some code here

Reply:
Done!

Reply:
Heya, I don't see this code in the blog post. Did you push?

Reply:
Wait, I do in the commit; maybe I didn't pull correctly.